doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -160,6 +166,12 @@ For maximum compatibility the named curve form should be used. Note: the
 B<OPENSSL_EC_NAMED_CURVE> value was only added to OpenSSL 1.1.0; previous
 versions should use 0 instead.
 
+The EVP_PKEY_CTX_set1_id(), EVP_PKEY_CTX_get1_id() and EVP_PKEY_CTX_get1_id_len()
+are used to manipulate special identifier field for specific signature algorithm
+such as SM2. The EVP_PKEY_CTX_get1_id_len() returns the length of the ID set in
+algorihtm related context. The length is usually used to allocate adequate memory
```

Comment:
I'd change this to "EVP_PKEY_CTX_get1_id_len() returns the length of the ID set via a previous call to EVP_PKEY_set1_id()"

Reply:
Good wording